### PR TITLE
Need help module

### PR DESCRIPTION
**Pull request by jefflembeck**
The meta package will largely be used as a set of components built out
of other components. This _does_ include a tiny bit of helper classes
though.
screen shot 2016-05-09 at 11 24 12 am
expanded

### CODE REVIEW

**Comment by aredridel** [comment]
Looks pretty good to me, but is `meta` the name we want?

(I ask because I made a `sidebars` as a start for the orgs features sidebar.)

**Comment by jefflembeck** [comment]
@rockbot nada, but that _does_ bring up that this relies on pui-css-bootstrap and a few other pieces for deps

**Comment by jefflembeck** [comment]
@aredridel @rockbot I am completely lacking opinion on what we should call this, but I can fold it into sidebars if that's what we want. I thought this was more generic than that.

**Comment by stubbornella** [comment]
I wouldn't call it sidebars, because we could have these in different places.

File: src/pivotal-ui/components/meta/meta.scss [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      </div>
+
+    </div>
+  </div>
+</div>
+
+```
+*/
+
+.need-help {
```
Comment (stubbornella) [inline src/pivotal-ui/components/meta/meta.scss:137]: do you need this? I think bootstrap has a specific class for this (and an animation if I'm not mistaken)

**Comment by aredridel** [comment]
Aright. Makes sense for me to rename that one then!

File: src/pivotal-ui/components/meta/meta.scss (hunk 1 above)
Comment (jefflembeck) [inline src/pivotal-ui/components/meta/meta.scss:137]: This is to make the button outright disappear when the button is clicked, unfortunately bootstrap just makes the text disappear, which leaves the button's wrapper still there.

Comment (stubbornella) [inline src/pivotal-ui/components/meta/meta.scss:137]: oh, nevermind, it hides the button. gotcha.

Comment (stubbornella) [inline src/pivotal-ui/components/meta/meta.scss:137]: lol, silly bootstrap. I wonder if they intended for you to put the class on the wrapper of the button? Say, a paragraph?

**Pull request merged**